disco_aws_automation/disco_datapipeline.py

```diff
@@ -0,0 +1,255 @@
+"""
+Back end behaviors for creating data pipelines.
```

Comment:
What's this? An entire new file with no unit tests?  Say it ain't so!

Yeah, it's a TODO.